syntax = "proto2";

message IncubationSessionCurrentTemperatureMessage {
    required float temperature = 2;
}
